agimus_controller/agimus_controller/ocp_param_base.py

```diff
@@ -1,19 +1,24 @@
 from dataclasses import dataclass, field
 
 
+@dataclass
+class DTFactorsNSeq:
+    """Time varying steps definition based on integration step of OCP."""
+
+    factors: list[int]  # Number of dts between two time steps, the "factor".
+    dts: list[int]  # Number of time steps, the "n".
```

Comment:
```suggestion
    n_steps: list[int]  # Number of time steps, the "n".
```
`dts` is misleading...

If renaming is too troublesome, you can simply add a `n_steps` property, with a clear comment saying that `dts` is a bad name.